internal/backends/mcu/renderer.rs

```diff
@@ -304,45 +299,53 @@ struct PrepareScene {
 }
 
 impl PrepareScene {
-    fn new(size: SizeF, scale_factor: ScaleFactor, default_font: FontRequest) -> Self {
+    fn new(size: PhysicalSize, scale_factor: ScaleFactor, default_font: FontRequest) -> Self {
         Self {
             items: vec![],
             rectangles: vec![],
             textures: vec![],
             state_stack: vec![],
             current_state: RenderState {
                 alpha: 1.,
-                offset: PointF::default(),
-                clip: RectF::new(PointF::default(), size / scale_factor.0),
+                offset: LogicalPoint::default(),
+                clip: LogicalRect::new(
+                    LogicalPoint::default(),
+                    size.cast() * scale_factor.inverse(),
```

Comment:
so size / scale_factor  didn't work?

Reply:
Hm, it didn't work earlier. But let me try again, I think I may have tried without the `.cast()`.